firefox_ui_tests/remote/security/manifest.ini

```diff
@@ -1,3 +1,5 @@
+[test_dv_certificate.py]
+[test_mixed_script_content_blocking.py]
```

Comment:
You will have conflicts anyway when you rebase that branch to master, once the other PR has been merged. Or maybe that one would be earlier. So if that would be the case this PR is blocked by the other PR.